Libraries/CommonFunctions.psm1

```diff
@@ -354,6 +354,12 @@ function Is-VmAlive {
         foreach ( $vm in $AllVMDataObject) {
             if ($global:IsWindowsImage) {
                 $port = $vm.RDPPort
+                if( $null -eq $port )
+                {
+                    # Note(seansp): When using heterogeneous tests, some VMs won't be using RDP. Allow these to validate using SSH.
+                    Write-LogInfo "RDP is <NULL> -- SSH is available and used instead."
+                    $port = $vm.SSHPort
```

Comment:
It can be generic
if ($vm.RDPPort) {
    $port = $vm.RDPPort
} else {
    $port = $vm.SSHPort
}

Reply:
Using generic check.  I did not remove the global:IsWindowsImage here.  It could be -that- generic ... but then we'd check for RDP every time and I don't think that's the intent.